workshops/gloo-mesh-demo/README.md

```diff
@@ -154,7 +154,7 @@ export PATH=$PWD/istio-${ISTIO_VERSION}/bin:$PATH
 istioctl version
 ```
 
-2. Install Istio to each of the remote clusters. If you're using local Kubernetes clusters on a Mac M1 or M2, use [these ARM instructions](problems-istio-arm.md) instead.
```

Comment:
If arm isnt needed anymore, we can delete the `problems-istio-arm.md` files as well as the istiooperator files for arm